qcfractal/tests/test_collections.py

```diff
@@ -564,7 +573,11 @@ def test_rectiondataset_dftd3_energies(reactiondataset_dftd3_fixture_fixture):
 def test_rectiondataset_dftd3_molecules(reactiondataset_dftd3_fixture_fixture):
     client, ds = reactiondataset_dftd3_fixture_fixture
 
-    mols = ds.get_molecules()
+    request_made = not ds._use_view(False)
+    ds._clear_cache()
+
+    with monitor_requests(client, "molecule", request_made=request_made):
```

Comment:
I didn't realize that monitor requests asserted as well. Could we change the name of this to `check_requests_monitor` and add error messages to the asserts so that it say something like "Made requests when none where expected!". Some sort of name/assert change in that category.